simple_static_website/input.tf

```diff
@@ -80,4 +80,10 @@ variable "force_destroy_s3_bucket" {
   description = "(Optional, default 'false') If true, the s3 bucket will be deleted even if it's full. Not advised for production use."
   type        = bool
   default     = false
+}
+
+variable "web_acl_arn" {
+  description = "(Optional, default '') ARN of the WAF Web ACL to associate with the CloudFront distribution (using version WAFv2)."
+  type        = string
+  default     = ""
```

Comment:
I think this might need to be a `null` default rather than a blank string.

Reply:
You are absolutely right Pat - it should be null. I have made the change. Thank you for noticing this.